### ISSUE TITLE
Dataset Download Module 

### ISSUE THREAD

**Issue by syasini** [syasini/AstroPaint#51]
Add a method for manually downloading the datasets in case git lfs is not installed. 

Use case example for the method: 

```python 
astropaint.download_dataset("WebSky_lite")
```

should download the `WebSky_lite.csv` file from [here](https://github.com/syasini/AstroPaint/tree/master/astropaint/data) to the local data directory. The [`path_dir`](https://github.com/syasini/AstroPaint/blob/master/astropaint/paint_bucket.py#L34) variable under `paint_bucket` can be used to find the local `data` directory` like this:

```python 
data_dir = os.path.join(path_dir, 'data')
```

other available methods include 

```python 
astropaint.download_dataset("All")
```

to download all the files under `data`.

**Comment by S-KSM** [comment]
v1 solution is going into the QA. One question, shall I put the function in the utils?

**Comment by syasini** [comment]
Thanks @S-KSM.  That's a great question! How about starting a new module called `datasets` and then calling the download function `download`? Then it will be used like this: 

```python 
from astropaint import datasets

datasets.download("WebSky_lite")
```

`datasets.py` can live under `astropaint/lib` and then added to the module namespace in `astropaint/__init__.py` as 

```python
from .lib import datasets
```

What do you think?

**Comment by syasini** [comment]
resolved in PR #60 and #61.
